Dockerfile

```diff
@@ -19,7 +21,7 @@ COPY . /work
 WORKDIR /work
 
 # Set up python env
-RUN pip install "poetry==1.4.0"
+RUN pip install "poetry==1.8.2"
```

Comment:
this is not entirely necessary, but the poetry in the project is at version 1.8.2, so it seems best to keep this up to date

Reply:
switched this up! I originally left it since the curl method was being a pain in recognizing that poetry was installed properly, but got it working now